packages/block-library/src/read-more/edit.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	InspectorControls,
+	RichText,
+	useBlockProps,
+} from '@wordpress/block-editor';
+import { ToggleControl, PanelBody } from '@wordpress/components';
+import { createBlock, getDefaultBlockName } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+export default function ReadMore( {
+	attributes: { content, linkTarget },
+	setAttributes,
+	insertBlocksAfter,
+} ) {
+	const blockProps = useBlockProps();
+	return (
+		<>
+			<InspectorControls>
+				<PanelBody title={ __( 'Link settings' ) }>
+					<ToggleControl
+						label={ __( 'Open in new tab' ) }
+						onChange={ ( value ) =>
+							setAttributes( {
+								linkTarget: value ? '_blank' : '_self',
+							} )
+						}
+						checked={ linkTarget === '_blank' }
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<RichText
+				tagName="a"
+				aria-label={ __( '"Read more" link text' ) }
+				placeholder={ __( 'Add "read more" link text' ) }
```

Comment:
Nit: I found this text to be very confusing. I actually went to the settings to check if there is a read more link text setting. This applies to the feature of the excerpt block as well. Can't we just default to a sensible choice, like "Read more", or "Permalink"?

Reply:
Yeah, it is confusing. We might be able to default to just "Read more", but I think we can probably explore that in a followup PR.

Reply:
Changed to `read more`. I'll land with green CI and let's iterate further.